intro_to_rl.ipynb

```diff
@@ -0,0 +1,3249 @@
+{
```

Comment:
<p>Same as excersise 1, print the hint</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>removed this</p>